lib/search.js

```diff
@@ -46,6 +34,9 @@ function search (args, silent, staleness, cb) {
 
   var searchopts = npm.config.get('searchopts')
   var searchexclude = npm.config.get('searchexclude')
+  if (npm.config.get('searchsort', 'cli')) {
+    log.warn('search', 'Search result sorting not currently supported')
+  }
```

Comment:
One option for searching that could work is to stream, but then stream the results to a full object or something, which could then be sorted and displayed. This will decrease the amount of memory required currently, but could still cause a heap blowup if the search term has a lot of results and the computer has a very small amount of RAM.